### ISSUE TITLE
Captive Core's run-from implementation doesn't work for ledgers < 63

### ISSUE THREAD

**Issue by 2opremio** [stellar/go#3155]
As part of #3153 I found out that run-from doesn't work for ledgers < 63. This is the error I am getting:

```
time="2020-10-23T10:08:46.126Z" level=error msg="Error in ingestion state machine" current_state="resume(latestSuccessfullyProcessedLedger=1)" error="error preparing range: opening subprocess: error calculating ledger and hash for stelar-core run: error trying to read ledger header 1 from HAS: error opening ledger stream: Bad HTTP response '404 File not found' for GET 'http://localhost:1570/ledger/00/00/00/ledger-00000001.xdr.gz'" next_state=start pid=194 service=ingest
```

Requesting `ledger/00/00/00/ledger-00000001.xdr.gz` isn't correct since it's not a checkpoint ledger.

**Comment by abuiles** [comment]
@2opremio this PR allows online mode to start from ledgers < 63 https://github.com/stellar/go/pull/3160

**Comment by 2opremio** [comment]
This is partly fixed by #3156 . It still doesn't work for ledger 1, but #3157 is already covering it. Closing
